argocd/aad-pod-identity/values.yaml

```diff
@@ -1,14 +1,21 @@
 ---
-# -- Values passed to the add-pod-identity chart
-aad-pod-identity:
+aad-pod-identity: 
+  # -- By default, AAD Pod Identity matches pods to identities across namespaces.
```

Comment:
I think we'd want to keep the top-level comment only here, because these sub-values are specific to the `aad-pod-identity` chart, and thus should not be documented in the DevOps Stack, (lest we start duplicating the docs for all the helm charts we use).

Reply:
Now fixed. I applied same principle for all charts